puppet/modules/web/manifests/jenkins.pp

```diff
@@ -1,25 +1,17 @@
-class web::jenkins(
+class web::jenkins (
   Stdlib::Fqdn $hostname = 'ci.theforeman.org',
-  Stdlib::Absolutepath $webroot = '/var/www/vhosts/jenkins/htdocs',
   Boolean $https = false,
 ) {
   include web::base
```

Comment:
`web::vhost` requires `web` which includes `web::base`

Reply:
I kept it in because we use `$apache::user` and `$apache::group` as docroot owners. I'm not sure we actually need that though.

Reply:
Dropped the docroot ownership.